lambdas/package/index.py

```diff
@@ -0,0 +1,186 @@
+"""
+Provide a virtual-file-system view of a package's logical keys.
+"""
+
+import io
+import json
+
+import boto3
+import pandas as pd
+
+from t4_lambda_shared.decorator import api, validate
+from t4_lambda_shared.utils import get_default_origins, make_json_response
+
+S3_DOMAIN_SUFFIX = '.amazonaws.com'
+
+SCHEMA = {
+    'type': 'object',
+    'properties': {
+        'bucket': {
+            'type': 'string'
+        },
+        'manifest': {
+            'type': 'string'
+        },
+        'access_key': {
```

Comment:
@nl0 is security any better if these secrets go in the headers instead of the query parameters? I am thinking of server logs.

Reply:
well, maybe the headers are a little less obvious, but this data gets exposed anyways, so i'm not sure we'll gain anything by using headers. tho it's quite easy to implement, so we can do it if you say so